core/controllers/editor.py

```diff
@@ -772,6 +772,10 @@ def post(self, exploration_id):
 
         fs.commit(self.user_id, filepath, raw, mimetype=mimetype)
 
+        if not feconf.DEV_MODE:
```

Comment:
This is in the wrong place. We should always create compressed versions. Just that in DEV_MODE we sub out the compressor in platform/ to just return a copy of the original image.

Reply:
Okay sure. Done!